src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java

```diff
@@ -273,8 +301,8 @@ void createConsumerWithForwardedHeaderWrongProto(VertxTestContext context) throw
                         assertTrue(ar.succeeded());
                         HttpResponse<JsonObject> response = ar.result();
                         HttpBridgeError error = HttpBridgeError.fromJson(response.body());
-                        assertEquals(HttpResponseStatus.INTERNAL_SERVER_ERROR.code(), response.statusCode());
-                        assertEquals(HttpResponseStatus.INTERNAL_SERVER_ERROR.code(), error.getCode());
+                        assertEquals(HttpResponseStatus.UNPROCESSABLE_ENTITY.code(), response.statusCode());
+                        assertEquals(HttpResponseStatus.UNPROCESSABLE_ENTITY.code(), error.getCode());
```

Comment:
This is testing for an invalid `proto` in the `Forwarded` _header_, which is not considered part of the request entity (=request body in my understanding), so I don't think 422 is correct in this case. See https://tools.ietf.org/html/rfc4918#section-11.2.

Reply:
Good point. The only status code that fits is just 400 bad request imho.